README.md

```diff
@@ -1,106 +1,122 @@
-# Wanderlust - Your Ultimate Travel Blog 🌍✈️
-
-WanderLust is a simple MERN travel blog website ✈ This project is aimed to help people to contribute in open source, upskill in react and also master git.
+<div>
+  <h1>Wanderlust</h1>
+  <h2>The Ultimate Travel Blog 🌍✈ for You </h2>
+</div>
 
 ![Preview Image](https://github.com/krishnaacharyaa/wanderlust/assets/116620586/17ba9da6-225f-481d-87c0-5d5a010a9538)
 
-## [Figma Design File](https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=0%3A1&mode=design&t=c4oCG8N1Fjf7pxTt-1)
-## [Discord Channel](https://discord.gg/FEKasAdCrG)
+<hr>
 
-## 🎯 Goal of this project
+<div>
+<h3>• Watch our collaboration video : </h3>
+  <a href="https://youtu.be/ANfC1u_N_A0?feature=shared"> <img src="https://github.com/krishnaacharyaa/wanderlust/assets/133582566/47d71dd6-0390-479e-9d4e-3f077ef1a987.png" width="45" height="40"></a> <h4> with TechwithShubham </h4> 
```

Comment:
Remove that dot and : from all the headers

Reply:
okk done